packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -19,6 +19,7 @@ import { focus, isTextField, placeCaretAtHorizontalEdge } from '@wordpress/dom';
 import { BACKSPACE, DELETE, ENTER } from '@wordpress/keycodes';
 import { __, sprintf } from '@wordpress/i18n';
 import { useSelect, useDispatch } from '@wordpress/data';
+import { BlockPropsFilterContext } from '@wordpress/blocks';
```

Comment:
This new exposed component bothers me a bit. I wonder if we can do without it.

Other than that, the PR looks good.

Reply:
Can we make this `__unstable`

Reply:
Yes, let's mark unstable.